extensions/bond/sawtooth_bond/updates/libor.py

```diff
@@ -114,7 +115,13 @@ class CreateLIBORUpdate(Update):
     # signed with the corresponding private key.
     __LIBOR_PUBLISHER_ADDR__ = '1UrR1WTfkMaWmY8z6DxwEE2MYFXA6rdzZ'
 
-    def __init__(self, update_type, date, rates, signature, object_id=None):
+    def __init__(self,
+                 update_type,
+                 date,
+                 rates,
+                 signature=None,
+                 LIBOR_public_key=None,
```

Comment:
@vaporos  and @feihujiang : I think @feihujiang made the "LIBOR" part all upper-case because of the fact that the acronym LIBOR is usually written in all upper-case (but not always). 

However, I think that it should be all lower-case, to follow the Python style guide recommendations.